src/auth/oauth2/qgso2.cpp

```diff
@@ -237,6 +237,15 @@ void QgsO2::link()
   }
 }
 
+
+void QgsO2::setState( const QString & )
```

Comment:
Should this have `const QString &value`. I mean, is `state` no longer user-configurable, or should it always just be a random number?

Reply:
correct: it's no longer user configurable: it's a nonce opaque value, automatically generated